bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html

```diff
@@ -0,0 +1,413 @@
+            online, cyberbullying and cyber violence threaten not just individuals, but
+            the Internet itself.
+          {% endtrans %}
+          </p>
+        </div>
+
+        <aside class="topic-intro-aside">
+          <div class="note">
+            <p>
+            {% trans %}
+              73% of Internet users have seen someone harassed online and 40% have personally experienced it.
+            {% endtrans %}
+            </p>
+            <p class="note-source">
+              <cite>
+              {% trans pew='http://www.pewinternet.org/2014/10/22/online-harassment/' %}
+                Source: <a href="{{ pew }}">Pew Research</a>, 2014
+              {% endtrans %}
+              </cite>
+            </p>
+          </div>
+          <span class="aside-accent" aria-hidden="true" role="presentation">73%</span>
+        </aside>
+      </div>
+    </section>
+
+    <section class="topic-action">
+      <div class="content">
+        <div class="topic-action-you">
+          <h3 class="topic-action-title">{{ _('How you can practice respect') }}</h3>
+          <ul>
+            <li>
+              <h4>{{ _('Report bullying') }}</h4>
+              <p>
+              {% trans report='http://cyberbullying.org/advice-for-adult-victims-of-cyberbullying' %}
+                If you see cyber violence and bullying online, <a href="{{ report }}">record it and report it</a>.
+              {% endtrans %}
+              </p>
+            </li>
+            <li>
+              <h4>{{ _('Talk about respect') }}</h4>
+              <p>
+              {% trans kids='http://www.connectsafely.org/tips-to-help-stop-cyberbullying/',
+                       teachers='https://d157rqmxrxj6ey.cloudfront.net/amira/33977/' %}
+                <a href="{{ kids }}">Talk to your kids</a> about cyberbullying, and <a href="{{ teachers }}">encourage their teachers to do the same</a>.
+              {% endtrans %}
+              </p>
+            </li>
+          </ul>
+        </div>
+
+        <div class="topic-action-moz">
+          <h3 class="topic-action-title">{{ _('How Mozilla is practicing respect') }}</h3>
+          <ul>
+            <li>
+              <h4>{{ _('Supporting tools for safety') }}</h4>
+              <p>
+              {% trans coral='https://coralproject.net/',
+                       talk='https://coralproject.net/products/talk.html' %}
+                Supporting the <a href="{{ coral }}">Coral Project</a>, which works to improve how journalists and
+                communities engage on news websites, including tools like <a href="{{ talk }}">Talk</a>, that aims
+                to create smarter, safer comments sections.
+              {% endtrans %}
+              </p>
+            </li>
+            <li>
+              <h4>{{ _('Empowering individuals') }}</h4>
+              <p>
+              {% trans teaching='https://d157rqmxrxj6ey.cloudfront.net/amira/33977/' %}
```

Comment:
Looks like it also lives at https://thimbleprojects.org/amira/33977/, which is much nicer. I'll update.